czicompress/tests/test_copyoperation.cpp

```diff
@@ -247,18 +247,14 @@ TEST_CASE("copyczi.2: run compression on simple synthetic document changes compr
   auto metadata_segment = reader_compressed_document->ReadMetadataSegment();
   auto metadata = metadata_segment->CreateMetaFromMetadataSegment();
 
-  auto compression_method = metadata->GetChildNodeReadonly("ImageDocument/Metadata/Information/Image/OriginalCompressionMethod");
-  std::wstring compression_method_string;
-  bool success = compression_method->TryGetValue(&compression_method_string);
-  REQUIRE(success == true);
-  REQUIRE(compression_method_string == std::wstring(L"Zstd1"));
-
-  std::wstring compression_level;
-  auto encoding_quality = metadata->GetChildNodeReadonly("ImageDocument/Metadata/Information/Image/OriginalEncodingQuality");
-  std::wstring encoding_quality_string;
-  success = encoding_quality->TryGetValue(&encoding_quality_string);
-  REQUIRE(success == true);
-  REQUIRE(encoding_quality_string == std::wstring(L"100"));
+  auto first_subblock = reader_compressed_document->ReadSubBlock(0);
+  auto compression_method = first_subblock->GetSubBlockInfo().GetCompressionMode();
+  REQUIRE(compression_method == libCZI::CompressionMode::Zstd1);
+
+  auto compression_parameters = metadata->GetChildNodeReadonly("ImageDocument/Metadata/Information/Image/CurrentCompressionParameters");
```

Comment:
Do we also want to make an assertion on the OrginalCompressionParameters?

Reply:
Done in update 4.